homeassistant/components/arcam_fmj/media_player.py

```diff
@@ -238,6 +243,45 @@ async def async_turn_off(self):
         """Turn the media player off."""
         await self._state.set_power(False)
 
+    async def async_browse_media(self, media_content_type=None, media_content_id=None):
+        """Implement the websocket media browsing helper."""
+        if media_content_id not in (None, "root"):
+            raise BrowseError(
+                f"Media not found: {media_content_type} / {media_content_id}"
+            )
+
+        presets = self._state.get_preset_details()
+
+        radio = [
+            {
+                "title": preset.name,
+                "media_content_id": f"preset:{preset.index}",
+                "media_content_type": MEDIA_TYPE_MUSIC,
+                "can_play": True,
+            }
+            for preset in presets.values()
+        ]
+
+        root = {
+            "title": "Root",
+            "media_content_id": "root",
+            "media_content_type": "library",
+            "can_play": False,
+            "children": radio,
+        }
+
+        return root
```

Comment:
After https://github.com/home-assistant/core/pull/39698 is merged, this should become 
```python
        return BrowseMedia(**root)
```

Reply:
Fixed it in 39598 instead